### PR TITLE
[DISCO-2370] Add pull request template to Contile

### PR DESCRIPTION
**Pull request by Trinaa**
## References

JIRA: [DISCO-2370](https://mozilla-hub.atlassian.net/browse/DISCO-2370)
GitHub: #534 

## Description

Proposal to add a PR template to this project. All ideas, modifications and suggestions welcome!

## PR Review Checklist

_Put an `x` in the boxes that apply_

- [ ] This PR conforms to the [Contribution Guidelines](https://github.com/mozilla-services/contile/blob/main/CONTRIBUTING.md)
- [ ] This PR conforms to the [Opsec policies](https://github.com/mozilla-services/websec-check)
- [ ] Functional and performance test coverage has been expanded and maintained 
- [ ] Documentation has been updated
- [ ] The PR title starts with the JIRA issue reference, format `[DISCO-####]`, and has the same title

[DISCO-2370]: https://mozilla-hub.atlassian.net/browse/DISCO-2370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by linabutler — APPROVED** [review]
Stamp!

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## References
+
+JIRA: [DISCO-TODO](https://mozilla-hub.atlassian.net/browse/DISCO-TODO)
+GitHub: [#TODO](https://github.com/mozilla-services/merino-py/issues/TODO)
```
Comment (pjenvey) [inline PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
GitHub: [#TODO](https://github.com/mozilla-services/contile/issues/TODO)
```

**Review by pjenvey — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Trinaa) [reply PULL_REQUEST_TEMPLATE.md:4]: 🤦🏻‍♀️ TY for the catch @pjenvey

**Comment by Trinaa** [comment]
@pjenvey Could you take another look?

**Pull request merged**